oddbird/oddleventy
# Upgrade accoutrement & docs, remove duplicate event

## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

![what are jellyfish made of?](https://images.unsplash.com/photo-1613516632788-ec29d8edbd60?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1234&q=80)

## Description
_The commit messages say what you did; this should explain why and/or how._

- remove duplicate web summit
- upgrade accoutrement
- update accoutrement docs

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- /authors/miriam/ to see events
- /accoutrement/docs/ to see 3.0 documentation
